web/packages/teleport/src/services/joinToken/types.ts

```diff
@@ -55,7 +55,7 @@ export type JoinRole =
 //   presenting a secret token.
 // - 'ec2' is a method where node will join with the EC2 join method
 // - 'iam' is a method where node will join with the IAM join method
-export type JoinMethod = 'token' | 'ec2' | 'iam';
+export type JoinMethod = 'token' | 'ec2' | 'iam' | 'github';
```

Comment:
Are these comments actually adding any value here ? I only ask since I saw this comment block when I was reviewing and it felt a bit redundant to me - Machine ID currently supports 9 join methods and this list is always growing. It's feasible that most of these will be supported by the frontend eventually and so far the comment for each join method here is just "'x' is a method where the node will join with the X join method" 😓

Reply:
Added the comment, but leaving the discussion open

Reply:
i agree with @strideynet, then lets remove the comment regarding ec2 iam and github